simulations/vip-245/simulations.ts

```diff
@@ -0,0 +1,76 @@
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import {
+  Speeds,
+  UNITROLLER,
+  VAI_VAULT_RATE,
+  blocksPerDay,
+  borrowSpeeds,
+  supplySpeeds,
+  vTokens,
+  vip245,
+} from "../../vips/vip-245";
+import COMPTROLLER_ABI from "./abi/Comptroller.json";
+
+const OLD_VAI_VAULT_RATE = parseUnits("125.00", 18).div(blocksPerDay);
+
+export const oldSupplySpeeds = Speeds.map(speed => speed.oldBorrowSpeed);
+export const oldBorrowSpeeds = Speeds.map(speed => speed.oldBorrowSpeed);
+
+forking(35518397, () => {
```

Comment:
I would update the block number to a value post VIP-240, when the XVS emission of TUSD were modified